File: JugnooNewDriver/jugnooNewDriver/src/main/java/product/clicklabs/jugnoo/driver/retrofit/model/drivertaks/Tasks.java
package product.clicklabs.jugnoo.driver.retrofit.model.drivertaks;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
/**
 * Awesome Pojo Generator
 * */
public class Tasks{
  @SerializedName("task_type")
  @Expose
  private int taskType;
  @SerializedName("advertise_credits")
  @Expose
  private double advertiseCredits;
  @SerializedName("start_time")
  @Expose
  private String startTime;
  @SerializedName("end_time")
  @Expose
  private String endTime;

  public int getTaskType() {
    return taskType;
  }

  public void setTaskType(int taskType) {
    this.taskType = taskType;
  }

  public double getAdvertiseCredits() {
    return advertiseCredits;
  }

  public void setAdvertiseCredits(double advertiseCredits) {
    this.advertiseCredits = advertiseCredits;
  }

  public String getStartTime() {
    return startTime;
  }

  public void setStartTime(String startTime) {
    this.startTime = startTime;
  }

  public String getEndTime() {
    return endTime;
  }

  public void setEndTime(String endTime) {
    this.endTime = endTime;
  }
}

File: JugnooNewDriver/jugnooNewDriver/src/main/java/product/clicklabs/jugnoo/driver/retrofit/model/drivertaks/Tasks.kt
package product.clicklabs.jugnoo.driver.retrofit.model.drivertaks

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Awesome Pojo Generator
 */
class Tasks {
    @SerializedName("task_type")
    @Expose
    var taskType: Int = 0
    @SerializedName("advertise_credits")
    @Expose
    var advertiseCredits: Double = 0.toDouble()
    @SerializedName("start_time")
    @Expose
    var startTime: String? = null
    @SerializedName("end_time")
    @Expose
    var endTime: String? = null
    @SerializedName("show_text")
    @Expose
    var message: String? = null
}